src/main/java/org/codewars/kata/implementation/Francious/SixImpl.java

```diff
@@ -24,7 +24,7 @@ public String balance(String book) {
     }
 
     public double f(double x) {
-        return 0;
+        return x / (Math.pow((1 + x), 0.5) + 1);
```

Comment:
we changed main idea to put in this class all gialogue with user about task. so print here result.
Should methods be void or Boolean for example, I think, we decide tomorrow